tests/python/pants_test/backend/python/tasks/test_interpreter_selection_integration.py

```diff
@@ -30,6 +30,40 @@ def hermetic(cls):
     # in the below tests.
     return True
 
+  def _build_pex(self, binary_target, config=None, args=None, version=PY_27):
```

Comment:
Simply moved the below 3 functions up so that they appear near the top of the class, in line with where we usually place util code in test classes.